.github/workflows/spark-ci.yml

```diff
@@ -141,4 +142,4 @@ jobs:
         with:
           name: test logs
           path: |
-            **/build/testlogs
\ No newline at end of file
+            **/build/testlogs
```

Comment:
nit: unnecessary change

Reply:
So I just tried removing the file changes from my local and the GH PR, then added this line again as I did in the other files and this artifact keeps popping up in the diff. Any thoughts on how to get rid of it? It's not a newline or even a space or anything. I tried those as well.

Reply:
Looks like the trailing newline was missing.